Remove coreapi requirement

## Description

The ``coreapi`` requirement should be removed from ``requirements/base.txt``.




## Rationale

From the [CoreAPI](https://github.com/core-api/core-api/) page:

>DEPRECATION NOTICE: As of djangorestframework 3.9, the plan is to phase out CoreAPI in favor of OpenAPI as the default schema representation. New projects should consider using OpenAPI rather than CoreAPI. See the DRF 3.9 release announcment for more details.

The [DRF 3.9 page](https://www.django-rest-framework.org/community/3.9-announcement/#whats-next) has an example of how to use the integrated OpenAPI support.


## Use case(s) / visualization(s)

None. It's a requirement for a module no longer developed. If it's used somewhere in the template that I've missed, it should be replaced with [OpenAPI](https://www.openapis.org/) equivalents.

I only noticed it because I'm going through a Django project generated by this cookiecutter and can't explain why the requirement exists.

Those instructions result in:

    AssertionError: `coreapi` must be installed for schema support.

And it looks like CoreAPI is still hardwired. So the question is now "Do you want a dependency that is an archived GitHub project with no developers, hoping that the DRF crew replace it someday, and even though the template does not even provide commented-out URLconfs or settings for anything DRF?"

--- a/CONTRIBUTORS.rst
+++ b/CONTRIBUTORS.rst
@@ -87,6 +87,7 @@ Listed in alphabetical order.
   Chris Pappalardo         `@ChrisPappalardo`_
   Christopher Clarke       `@chrisdev`_
   Cole Mackenzie           `@cmackenzie1`_
+  Cole Maclean             `@cole`_                      @cole
   Collederas               `@Collederas`_
   Craig Margieson          `@cmargieson`_
   Cristian Vargas          `@cdvv7788`_
@@ -258,6 +259,7 @@ Listed in alphabetical order.
 .. _@chuckus: https://github.com/chuckus
 .. _@cmackenzie1: https://github.com/cmackenzie1
 .. _@cmargieson: https://github.com/cmargieson
+.. _@cole: https://github.com/cole
 .. _@Collederas: https://github.com/Collederas
 .. _@curtisstpierre: https://github.com/curtisstpierre
 .. _@dadokkio: https://github.com/dadokkio

--- a/{{cookiecutter.project_slug}}/requirements/base.txt
+++ b/{{cookiecutter.project_slug}}/requirements/base.txt
@@ -31,4 +31,3 @@ django-redis==4.11.0  # https://github.com/niwinz/django-redis
 
 # Django REST Framework
 djangorestframework==3.11.0  # https://github.com/encode/django-rest-framework
-coreapi==2.3.3  # https://github.com/core-api/python-client

